models/repo/user_repo.go

```diff
@@ -170,3 +170,26 @@ func GetReviewers(ctx context.Context, repo *Repository, doerID, posterID int64)
 	users := make([]*user_model.User, 0, 8)
 	return users, db.GetEngine(ctx).Where(cond).OrderBy(user_model.GetOrderByName()).Find(&users)
 }
+
+// GetPullRequestPosters returns all users that have authored a pull request for the given repository
+func GetPullRequestPosters(ctx context.Context, repo *Repository) (_ []*user_model.User, err error) {
```

Comment:
Either use named results or don't use them. I'm personally more a fan of no named results as they are bit confusing sometimes.

Reply:
Sure thing. This signature was copied from GetRepoAssignees() on line 61 of this same file, but I can make the change.

Reply:
Hmm I see, yeah only using named results for one of the arguments is just *useless* IMO.

Reply:
Done. cacf5791576bf60f79421e8a2fc42ecc69caf1e3